### PR TITLE
Add terminal width fallback via stty if on windows/MSYS2

### PR DESCRIPTION
**Pull request by th1000s**

### CODE REVIEW

**Comment by th1000s** [comment]
I could finally verify that this workaround fixes the problem on Windows.

Also added a `DELTA_NO_WORKAROUNDS` env var to make future debugging easier, e.g. when eminence/terminal-size#25 is fixed.

File: src/utils/workarounds.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    match (height, width) {
+        (24..=25, 79..=80) => guess_real_width(width, term_stdout).unwrap_or(width),
+        _ => width,
+    }
+    .into()
+}
+
+#[cfg(not(target_os = "windows"))]
+pub fn windows_msys2_width_fix(height_width: (u16, u16), _: &console::Term) -> usize {
+    let _ = NO_WORKAROUNDS;
```
Comment (dandavison) [inline src/utils/workarounds.rs:58]: What's this for?

Comment (th1000s) [reply src/utils/workarounds.rs:58]: Just so this const &str does not trigger an unused variable warning in the non-windows case.

Comment (dandavison) [reply src/utils/workarounds.rs:58]: Ah of course, thanks.

**Comment by dandavison** [comment]
Thank you @th1000s.

**Pull request merged**